router/composes.go

```diff
@@ -0,0 +1,10 @@
+package router
+
+import (
+	"github.com/labstack/echo/v4"
+)
+
+func getComposersHandler(c echo.Context) error {
+	 return nil
+
+}
```

Comment:
ファイルの最後は改行入れてください:metal:
```suggestion
}

```

Reply:
完了